config/locales/static/terms_and_conditions.yml

```diff
@@ -4,189 +4,160 @@ en:
-
-        ## Linking to Child development training
-
-        We welcome and encourage other websites to link to Child development training.
-
-        You must [contact us](mailto:child-development.training@education.gov.uk?subject=Request%20permission){:class="govuk-link"}
-        for permission if you want to say your website is associated with or 
-        endorsed by Child development training or another government department 
-        or agency.
-
-        ## Linking from Child development training
-
-        Child development training links to websites that are managed by other 
-        government departments and agencies, service providers or other 
-        organisations. We do not have any control over the content on these 
-        websites.
-
-        We're not responsible for:
-
-        * the protection of any information you give to these websites
-        * any loss or damage that may come from your use of these websites, or 
-        any other websites they link to
-
-        You agree to release us from any claims or disputes that may come from 
-        using these websites.
+        ##Personal and sensitive information
+        
+        Child development training includes a learning log for you to make notes. In these notes, you must not enter any personal or sensitive information about: 
 
-        You should read all terms and conditions, privacy policies and end user 
-        licences that relate to these websites before you use them.
+        - children
+        - parents
+        - colleagues
 
-        ## Child development training content
+        Under the UK General Data Protection Regulation (GDPR), sensitive personal information includes information about someone’s: 
 
-        We update Child development training all the time. We can change or 
-        remove content at any time without notice.
+        - race or ethnic origin 
+        - political, religious or philosophical beliefs 
+        - trade union membership
+        - genetic or biometric data 
+        - health 
+        - sex life or sexual orientation
 
-        The content on Child development training is the intellectual property 
-        of the DfE. You must not copy or reproduce this content for any purpose. 
+        You also must not record information that could be used to identify someone, such as:
 
-        ## Disclaimer
+        - names
+        - where they live 
+        - what they look like
 
-        While we make every effort to keep Child development training up to date, 
-        we do not provide any guarantees, conditions or warranties that the 
-        information will be:
+        You may need to include relevant details about a child, for example that they live with their grandma, or that they have a developmental delay. This is fine, as long as you do not include any other details which could link to the child. You could use pseudonyms like “child A” in your notes. 
```

Comment:
Does it need to be a capital `Grandma`?

Reply:
Nope, not in this case. Only if you are using it as their name I think e.g.: 

'How are you feeling, Grandma?' she asked.

Reply:
Proper noun 👍🏼